### PR TITLE
Staging to Production

### PR DESCRIPTION
**Pull request by andela-oakinwa**
#### What does this PR do 
- Updates the `master` branch with `staging`

#### Description of Task to be completed
- Merge branch
- Complete API routes

### CODE REVIEW

**Comment by andela-rakpan** [comment]
@andela-oakinwa Ignore your `coverage` folder in your `.gitignore` file

File: .codeclimate.yml [hunk 1]
```diff
@@ -27,4 +27,7 @@ ratings:
 exclude_paths:
 - node_modules/
 - server/tests/
-- out/
\ No newline at end of file
+- out/
+- coverage/
+- docs/
+- db/
```
Comment (andela-rakpan) [inline .codeclimate.yml]: Put a line-space here

File: .eslintignore [hunk 2]
```diff
@@ -1,4 +1,5 @@
 **/*{.,-}min.js
 webpack.config.js
 karma.conf.js
-out/
\ No newline at end of file
+out/
+server/tests
```
Comment (andela-rakpan) [inline .eslintignore]: Put a line-space here

File: Procfile [hunk 3]
```diff
@@ -1 +1 @@
-web: npm run server
\ No newline at end of file
+web: npm run start
```
Comment (andela-rakpan) [inline Procfile]: Put a line-space after your last line and in all occurrences.

File: .codeclimate.yml (hunk 1 above)
Comment (andela-oakinwa) [inline .codeclimate.yml]: Fixed.

File: Procfile (hunk 3 above)
Comment (andela-oakinwa) [inline Procfile]: Fixed

File: .eslintignore (hunk 2 above)
Comment (andela-oakinwa) [inline .eslintignore]: Fixed

File: testSetup.js [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+// Register babel so that it will transpile ES6 to ES5
+// before our tests run.
+require('babel-register')();
+
+// Disable webpack-specific features for tests since
+// Mocha doesn't know what to do with them.
+require.extensions['.css'] = function () { return null; };
+require.extensions['.png'] = function () { return null; };
+require.extensions['.jpg'] = function () { return null; };
```
Comment (andela-oakinwa) [inline testSetup.js]: File ignored

File: testSetup.js [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+process.env.NODE_ENV = 'test';
+
+// Register babel so that it will transpile ES6 to ES5
+// before our tests run.
+require('babel-register')();
+
+// Disable webpack-specific features for tests since
+// Mocha doesn't know what to do with them.
+require.extensions['.css'] = function () { return null; };
+require.extensions['.png'] = function () { return null; };
```
Comment (andela-oakinwa) [inline testSetup.js]: File ignored

File: testSetup.js [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+// hot module reloading code) doesn't apply for tests.
+process.env.NODE_ENV = 'test';
+
+// Register babel so that it will transpile ES6 to ES5
+// before our tests run.
+require('babel-register')();
+
+// Disable webpack-specific features for tests since
+// Mocha doesn't know what to do with them.
+require.extensions['.css'] = function () { return null; };
```
Comment (andela-oakinwa) [inline testSetup.js]: File ignored

**Pull request merged**